mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraModSeqProvider.java

```diff
@@ -167,7 +168,8 @@ public long highestModSeq(MailboxSession mailboxSession, MailboxId mailboxId) th
             update.bind()
                 .setUUID(MAILBOX_ID, mailboxId.asUuid())
                 .setLong(NEXT_MODSEQ, nextModSeq.getValue())
-                .setLong(MOD_SEQ_CONDITION, modSeq.getValue()))
+                .setLong(MOD_SEQ_CONDITION, modSeq.getValue())
+                .setConsistencyLevel(ConsistencyLevel.SERIAL))
```

Comment:
Does the update use the read or write consistency level ?
https://docs.datastax.com/en/archived/cassandra/3.0/cassandra/dml/dmlConfigConsistency.html

I didn't find this information.
if it's the write, the serial consistency level is not listed ?